.envrc

```diff
@@ -1 +1,2 @@
 layout_python python3
+export PRINT_OUTPUT_FILE="/Users/ivan/dev/py/librocco/receipt-printer/prints/dev-printer"
```

Comment:
I propose we change this line to `/tmp/receipt_printer` and you run `ln -s /tmp/receipt_printer /Users/ivan/dev/py/librocco/receipt-printer/prints/dev-printer` so this setup can work on other people's machines too.

Reply:
Yeeah...forgot about that